src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,107 @@
+import {concat, from, fromEvent} from 'rxjs';
+import {flatMap, filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+import {deconstructHydraId} from '@webex/common';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+
+const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @private
+   * @param {string} status
+   * @returns {string} PersonStatus
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Fetches the person data from the sdk and returns in the shape required by adapter.
+   *
+   * @private
+   * @param {string} ID  ID of the person for which to fetch data
+   * @returns {Person}
+   * @memberof PeopleSDKAdapter
+   */
+  async fetchPerson(ID) {
+    const {id, emails, displayName, firstName, lastName, avatar, orgId} = await this.datasource.people.get(ID);
+
+    return {
+      ID: id,
+      emails,
+      displayName,
+      firstName,
+      lastName,
+      avatar,
+      orgID: orgId,
+    };
+  }
+
+  /**
+   * Returns an observable that emits person data of the given ID.
+   *
+   * @public
+   * @param {string} ID ID of person to get.
+   * @returns {Observable.<Person>}
+   * @memberof PeopleSDKAdapter
+   */
+  getPerson(ID) {
+    if (!(ID in this.getPersonObservables)) {
+      const personUUID = deconstructHydraId(ID).id;
+      const person$ = from(this.fetchPerson(ID));
+
+      // Subscribe to `Apheleia` internal service to listen for presence changes
```

Comment:
Let's keep wording consistent
```suggestion
      // Subscribe to `Apheleia` internal service to listen for status changes
```